src/subscription_manager/repolib.py

```diff
@@ -167,6 +174,11 @@ def enable_pkg_plugins(cls):
         :return: It returns list of enabled plugins
         """
 
+        d_name, d_version, d_id, vers_mod, id, id_like = HardwareCollector().get_distribution()
+
+        if id is "debian" or id_like is "debian":
+            return []
+
```

Comment:
Also, better move this check _after_ the `is_auto_enable_enabled()` check: that one is way less expensive, so in case `rhsm.auto_enable_yum_plugins` is disabled then there is no need to check the distribution.